src/ClientResolver.php

```diff
@@ -281,6 +281,20 @@ class ClientResolver
             'doc'       => 'Set to false to disable checking for shared aws config files usually located in \'~/.aws/config\' and \'~/.aws/credentials\'.  This will be ignored if you set the \'profile\' setting.',
             'default'   => true,
         ],
+        'ignore_configured_endpoint_urls' => [
+            'type'      => 'value',
+            'valid'     => ['bool'],
+            'doc'       => 'Set to true to disable endpoint urls configured using the `endpoint_url` configuration option.',
+            'fn'        => [__CLASS__, '_apply_ignore_configured_endpoint_urls'],
+            'default'   => [__CLASS__, '_default_ignore_configured_endpoint_urls'],
+        ],
+        'endpoint_url' => [
```

Comment:
Since this does the same thing as endpoint, why not just have endpoint look in the config variables and environment variables?

Reply:
To ensure consistency in naming across all sources.

Reply:
Alternate option, we could add an alias field and a little logic in the resolver code so that it gets taken care of there